### DISCUSSION TITLE
Simple Gotek usage guide

### DISCUSSION THREAD

**Discussion by mjmonjure** [keirf/flashfloppy#772] [General]
Is there a simple guide which explains how to use Flash Floppy menu (3.38).  Like how to setup files for loading, how to bypass the menu altogether and boot from the hard drive etc., etc, etc.  I cannot find instructions anywhere.  There are plenty of videos, but they are dated and do not include instructions for Goteks with rotary dials.  Example, can I plug my flash drive into the Gotek after it's booted into the hard drive and access floppy images.  I have so many questions.

**Comment by keirf** [comment]
I can at least say that yes it is fine to hotplug the usb drive. So you can boot with no usb drive inserted.
